### ISSUE TITLE
xunit.runner.msbuild 2.3 throws error MSB4018

### ISSUE THREAD

**Issue by martinRocks** [xunit/xunit#1527]
When using xUnit msBuild running 2.2, I can run tests normally.  However, when I upgrade to 2.3, I get the below error.

`The "xunit" task failed unexpectedly.
System.ArgumentNullException: Value cannot be null.
Parameter name: input
	at System.Xml.XmlReaderSettings.CreateReader(Stream input, Uri baseUri, String baseUriString, XmlParserContext inputContext)
	at System.Xml.XmlReader.Create(Stream input, XmlReaderSettings settings, String baseUri)
	at Xunit.Runner.MSBuild.CrossPlatform.Transform(IRunnerLogger logger, String outputDisplayName, String resourceName, XNode xml, ITaskItem outputFile) in C:\Dev\xunit\xunit\src\xunit.runner.msbuild\Utility\CrossPlatform.cs:line 16
	at Xunit.Runner.MSBuild.xunit.Execute() in C:\Dev\xunit\xunit\src\xunit.runner.msbuild\xunit.cs:line 212
	at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
	at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()`

**Comment by bradwilson** [comment]
Please provide a repro.

**Comment by martinRocks** [comment]
I made a quick little demo of the problem.  It is made with VS 2017 v15.4.1.

There are 2 projects in the solution; a class library with one method and a class library for the test.  In the test project, I have added the below to run the test.

`  <Target Name="Test">
    <xunit Assemblies="bin\$(Configuration)\$(AssemblyName).dll" nunit="..\XunitErrorDemo_Dll.xml" />
  </Target>`

I have included a batch file of exactly what I'm doing to run the tests with msbuild.  I haven't included any packages.

Thank you for looking

[XunitErrorDemo.zip](https://github.com/xunit/xunit/files/1411005/XunitErrorDemo.zip)
